tools/ci/ci_wptrunner_infrastructure.sh

```diff
@@ -12,19 +12,23 @@ test_infrastructure() {
     else
         ARGS=$1
     fi
-    TERM=dumb ./wpt run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $ARGS $PRODUCT infrastructure/
+    TERM=dumb ./wpt $2 run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $ARGS $PRODUCT infrastructure/
```

Comment:
Nit; for documentation purposes, can you name this variable just so its a bit clearer? E.g.

```
PY3_FLAG="$2"
TERM=dumb ./wpt $PY3_FLAG run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts $ARGS $PRODUCT infrastructure/

```